cmd/remoteva/main.go

```diff
@@ -18,6 +19,24 @@ import (
 type Config struct {
 	RVA struct {
 		vaConfig.Common
+
+		// ServerUseAlternateClientAuthPolicy, when disabled, will cause the
```

Comment:
Maybe we could call this VerifyGRPCClientCertIfGiven?

Reply:
That name works too. Yours is definitely more descriptive of what the "alternate client auth policy" is.